templates/emails/email-addresses.php

```diff
@@ -24,11 +24,11 @@
 
 ?><table id="addresses" cellspacing="0" cellpadding="0" style="width: 100%; vertical-align: top; margin-bottom: 40px; padding:0;" border="0">
 	<tr>
-		<td style="text-align:<?php echo $text_align; ?>; font-family: 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif; border:0; padding:0;" valign="top" width="50%">
-			<h2><?php _e( 'Billing address', 'woocommerce' ); ?></h2>
+		<td style="text-align:<?php echo esc_html( $text_align ); ?>; font-family: 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif; border:0; padding:0;" valign="top" width="50%">
```

Comment:
`text-align:` should use `esc_attr()`, since is a attribute, `esc_html()` is not safe enough here.